boardswarm-cli/src/main.rs

```diff
@@ -430,6 +430,7 @@ enum DeviceCommand {
 
 #[derive(Debug, Subcommand)]
 enum RockCommand {
+    /// Transfer concatenated bootrom images of type 0x471 and 0x472 to a rock device
```

Comment:
it's not a concatenation of bootrom images though; it's a combined boot file for which the exact rockchip jargon unfortunately escapes me

Reply:
Let's then try: "Transfer a boot file to a rock device"